### ISSUE TITLE
RBL Kelompok 7

### ISSUE THREAD

**Issue by dudung** [dudung/fi1101-04-2022-1#25]
# Kelompok 7

No | NIM | Nama | GitHub
:-:  | :-: | :-: |  :-:
1 |  |  | 
2 |  |  | 
3 |  |  | 
4 |  |  | 
5 |  |  | 
6 |  |  | 
7 |  |  | 
8 |  |  | 

## Tugas 16-nov-2022
+ Lengkapi tabel di atas dengan mengisikan NIM, Nama dan user GitHub (disertai tautannya).
+ Selanjutnya diskusi mengenai perkembangan RBL dituliskan di issue ini sehingga dapat terpantau aktivitasnya.

**Comment by windavellisa** [comment]
Winda Vellisa Virgiani 
16022304

**Comment by Kamiliatazkia** [comment]
Kamilia Tazkia Rahma Andini
16022466

**Comment by NailaFazaChoirunissa** [comment]
Naila Faza Choirunissa
16022286

**Comment by afifahulya** [comment]
Afifah Ulya Nadira
16022454

**Comment by hasnanaa** [comment]
Hasna Rosyida Nur Adila
16022046

**Comment by nauradhiy** [comment]
Naura Dhia Inayah
16022094

**Comment by windavellisa** [comment]
@NailaFazaChoirunissa Semangat RBL

**Comment by izzanasira** [comment]
Izza Nasira Azzahra
16022004

**Comment by lintangsemestaa** [comment]
Lintang Arian Semesta
16022424

**Comment by windavellisa** [comment]
JOIN GRUP : 

+ [link](https://chat.whatsapp.com/BmJtnjcEm7oAg93uRt2X5J)

**Comment by windavellisa** [comment]
A | B | C 
:-:|:-:|:-:|
1 | 2 | 3 
4 | 5 | 6

**Comment by NailaFazaChoirunissa** [comment]
A | B | C
:-:|:-:|:-:|
1 | 2 | 3
4 | 5 | 6

**Comment by Kamiliatazkia** [comment]
A | B | C
:-:|:-:|:-:|
1 | 2 | 3
4 | 5 | 6

**Comment by windavellisa** [comment]
NIM | NAMA
:-:|:-:| 
004 | Izza 
046 | Hasna
094 | Naura 
286 | Naila 
304 | Winda 
424 | Lintang 
454 | Afifah 
466 | Kamilia

**Comment by windavellisa** [comment]
$$ 
y = mx + c 
$$

**Comment by NailaFazaChoirunissa** [comment]
$$
y=ax^2+bx+\frac{1}{y}
$$

**Comment by nauradhiy** [comment]
```mermaid
flowchart
    A-->B
```

**Comment by izzanasira** [comment]
`Kerkom_01` 
 
Hari, tgl : Jumat, 18 November 2022
Waktu : 19.00 - selesai
Tempat : Via zoom 
Link : 
https://itb-ac-id.zoom.us/j/92165787023?pwd=Q1FnYkxlbEsxdHBsSXFpUlJUVk1RZz09

Meeting ID: 921 6578 7023
Passcode: 690720

Agenda : 
Diskusi tentang alat dan bahan yang digunakan untuk merakit mobil pendulum

**Comment by dudung** [comment]
Tolong direkam atau di-screen shot, ya. Gambarnya nanti bisa dijadikan bagian dari laporan dan video. Terimakasih, @izzanasira .

**Comment by izzanasira** [comment]
> Tolong direkam atau di-screen shot, ya. Gambarnya nanti bisa dijadikan bagian dari laporan dan video. Terimakasih, @izzanasira .

Baik pak🙏

**Comment by izzanasira** [comment]
`Hasil_Kerkom`

Rencana Pelaksanaan :
- Hari, tgl : Selasa, 22 November 2022
- Waktu : 07.00 - selesai
- Tempat : TVST

Rencana Bahan
- Solasi 4 : naila
- Tutup botol : hasna
- Tusuk sate : naila
- Sedotan aqua : naila
- Alas : naura
- Sumpit : naila
- Gabus ijo : pepel
- Plastisin : pepel
- Kawat : pepel
- Cutter : all
- Gunting : all
- Lem tembak : naila
- Penggaris : all
- Busur : all
- Tang : pepel
- Sedotan gede : hasna
- Hvs : all
- Double tipe : all
- Kabel : pepel
- Batre 8 : pepel
- Timbangan : pepel

**Comment by izzanasira** [comment]
`Dokumentasi`
![IMG-20221118-WA0018](https://user-images.githubusercontent.com/112610339/202720606-a9816959-0c62-4d0f-8a12-db5077e33d57.jpg)
Kerkom 1 - Jumat, 18 November 2022

**Comment by afifahulya** [comment]
makasih zaaa

**Comment by afifahulya** [comment]
makasi semuaa

**Comment by lintangsemestaa** [comment]
makasiih temen-temen. makasih izzzaaa

**Comment by nauradhiy** [comment]
makasiih temen-temen

**Comment by NailaFazaChoirunissa** [comment]
terimakasihh guyss

**Comment by windavellisa** [comment]
Terima kasih all @lintangsemestaa @izzanasira @afifahulya @Kamiliatazkia @nauradhiy @NailaFazaChoirunissa @hasnanaa

**Comment by dudung** [comment]
Good progress. 👍

**Comment by windavellisa** [comment]
PROGRESS 2 : 

![20221122_091453](https://user-images.githubusercontent.com/112611172/203201316-bbf5e8e7-21fb-4ced-8087-696fa288f0cd.jpg)

**Comment by Kamiliatazkia** [comment]
![20221122_135834](https://user-images.githubusercontent.com/112610422/203246563-c445a90d-2498-479a-a285-f237ab372c61.jpg)
Lapor pak kita lg ambil data 😊

**Comment by dudung** [comment]
Bila ada videonya? Sebaiknya diunggah ke YouTube dan nanti dilink ke sini. Terima kasih sebelumnya.

**Comment by windavellisa** [comment]
> Bila ada videonya? Sebaiknya diunggah ke YouTube dan nanti dilink ke sini. Terima kasih sebelumnya.

Ada videonya pak, tapi masih proses pengeditan

**Comment by windavellisa** [comment]
**Berikut data pengamatan dari mobil bertenaga pendulum kami, mohon koreksinya pak😊🙏**

![Screenshot (24)](https://user-images.githubusercontent.com/112611172/203445815-c77c3178-ff18-4703-bb2c-5b0ed28e91b3.png)
![Screenshot (25)](https://user-images.githubusercontent.com/112611172/203445827-ea1907da-04b9-48d1-b13d-5cd03b0401e5.png)
![Screenshot (26)](https://user-images.githubusercontent.com/112611172/203445835-5920ac36-3c97-4e43-a524-b401dd2166fe.png)

**Comment by dudung** [comment]
Terima kasih, @windavellisa. Tolong dilengkapi sesuai [informasi tadi](https://github.com/dudung/fi1101-04-2022-1/blob/main/note/14/text/pengambilan-data-rbl.md), ya.

**Comment by lintangsemestaa** [comment]
@dudung Permisi, Pak. Mohon maaf mengganggu waktunya 🙏🏻 Untuk bagian analisis, apa saja yang harus dibahas? Penurunan rumus apa saja yang harus dicantumkan? Terima kasih, Pak🙏🏻

**Comment by NailaFazaChoirunissa** [comment]
![20221128_193750.jpg](https://user-images.githubusercontent.com/112611083/204280086-e4367f4d-d1f3-44b4-9303-c2b1a9d006c1.jpg)

—Kerja Kelompok RBL ke-3

**Comment by Kamiliatazkia** [comment]
Selamat malam pak, izin bertanya apakah jumlah halaman untuk laporan rbl ini hanya boleh 10 halaman? Apakah boleh lebih dari 19 halaman? Karena grafik dan tabel cukup memakan tempat dalam laporan pak 🙏🏻 terima kasih pak🙏🏻

**Comment by dudung** [comment]
> Selamat malam pak, izin bertanya apakah jumlah halaman untuk laporan rbl ini hanya boleh 10 halaman? Apakah boleh lebih dari 19 halaman? Karena grafik dan tabel cukup memakan tempat dalam laporan pak 🙏🏻 terima kasih pak🙏🏻

Laporan utama hanya 10 halaman. Tabel dan grafik pilih satu variasi saja, sisanya pindahkan ke lampiran. Lampiran silakan banyak halamannya.

**Comment by izzanasira** [comment]
Terima kasih bapak

**Comment by izzanasira** [comment]
Izin bertanya pak, untuk video presentasinya apakah besok ditampilkan di kelas?

**Comment by dudung** [comment]
> Izin bertanya pak, untuk video presentasinya apakah besok ditampilkan di kelas?

Iya, @izzanasira. Dan tolong sampaikan juga kelompok lain, ya. Terima kasih.

**Comment by izzanasira** [comment]
Baik pak

**Comment by izzanasira** [comment]
`Video` https://www.youtube.com/watch?v=HD5LZxnsWhY

**Comment by izzanasira** [comment]
`Laporan` [RBL Gerobak Bertenaga Pendulum Kelompok 7.pdf](https://github.com/dudung/fi1101-04-2022-1/files/10114321/RBL.Gerobak.Bertenaga.Pendulum.Kelompok.7.pdf)

**Comment by dudung** [comment]
Good progress, @izzanasira dan tim 👍

**Comment by hasnanaa** [comment]
Makasih zaa
